experimental/dds/tree2/src/test/feature-libraries/editable-tree-2/utils.ts

```diff
@@ -89,5 +89,14 @@ export function itWithRoot<TRoot extends TreeFieldSchema>(
  * Similar to JSON stringify, but preserves `undefined` and numbers numbers as-is at the root.
  */
 export function pretty(arg: unknown): number | undefined | string {
-	return arg === undefined ? "undefined" : typeof arg === "number" ? arg : JSON.stringify(arg);
+	if (arg === undefined) {
+		return "undefined";
+	}
+	if (typeof arg === "number") {
+		return arg;
+	}
+	if (typeof arg === "string") {
+		return `"${arg}"`;
+	}
```

Comment:
This is the new feature here; wanted to see a "-3" rendered differently than a -3 in the new tests.

Reply:
Makes total sense.